File: spring/src/main/java/study/hello/Greeting.java
package study.hello;

public class Greeting {
    private String name;
    private String content;

    public Greeting(String name, String content) {
        this.name = name;
        this.content = content;
    }

    public String getName() {
        return name;
    }

    public String getContent() {
        return content;
    }
}

File: springkt/src/main/kotlin/me/sshplendid/study/springkt/api/greeting/Greeting.kt
package me.sshplendid.study.springkt.api.greeting

import java.time.LocalDateTime
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
data class Greeting (
    @Id @GeneratedValue
    val id: Int,
    val name: String = "World",
    val content: String,
    val sendDateTime: LocalDateTime = LocalDateTime.now()
)
